apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/options/malOptions.tsx

```diff
@@ -346,3 +366,7 @@ const updateData = (data: any, initalValues: any) => {
 	}
 	return newData
 }
+
+const isNullfelt = (data: any) => {
+	return !data || data?.length < 1
+}
```

Comment:
Tror denne kan erstattes med lodash sin isEmpty, så slipper vi en egen metode for denne sjekken 🤓

Reply:
Aaa, den hadde jeg glemt - nice! 🤓